static/src/javascripts/polyfill.io

```diff
@@ -0,0 +1 @@
+https://polyfill.guim.co.uk/v2/polyfill.min.js?features=es6,es7,default-3.6&flags=gated&callback=guardianPolyfilled
```

Comment:
@phamann There was no major reason for doing this other than having a clean slate as a separate service in Fastly where we could monitor the affect of having ESIs. If I did it on `assets.guim.co.uk` I would have had to turn on ESIs there and would have found it harder to see the efficiency of the cache with all the other assets mixed in.